test/functional/test_site/expected/test_md_fragment.html

```diff
@@ -1,40 +1,32 @@
 <!DOCTYPE html>
 <html>
-
 <head>
-  <meta name="default-head-top">
```

Comment:
Seems like some of the calls are actually necessary; Likely the two that follow `this.pageSections[...] = htmlBeautify(...`

Reply:
I'm not sure I understand.
The effect of `this.pageSections[...] = htmlBeautify(... ` seems to be that beautification is only done on sections like `<nav>`, `<header>`, `<footer>` and  `#site-nav`. 
It will not result in `<head>` beautification.
All these sections (mentioned above) will anyway be beautified as they are part of the main HTML.

Or is it that we _want_ beautification of these sections by default, and only allow the main content (as in the `.md` files excluding the options in frontmatter) to be beautified through  the toggle?

Reply:
Placed the comment at a bad location, see https://github.com/MarkBind/markbind/pull/1116#discussion_r395998678

Reply:
Might not be an issue with `this.pageSections[...] = htmlBeautify(...` as well, it seems to occur in quite a few areas 😕